server/WEBHOOKS.md

```diff
@@ -230,6 +232,7 @@ function when comparing signatures.
 ```
 
 - <a id="comment-created-event">**COMMENT_CREATED**</a>
+  - only sent for published comments (ie status NONE or APPROVED)
```

Comment:
decided to add these to the doco to clarify how these events work ... as we were caught out expecting these to be triggered for all comments

Reply:
Thank you! We will probably also want to update our `webhooks.md` in `docs`.

Reply:
Ahh, I didn't realise there was another file. Let me apply the same change there.

Reply:
Have now applied the same change to docs/docs/webhooks.md